inputs/manual_input.csv

```diff
@@ -55,12 +55,18 @@ CH4 (g) submarine pipeline,investment,2002,112.64,EUR/MW/km,2014,Kaiser (2017):
-H2 (g) submarine pipeline,investment,2002,322.91,EUR/MW/km,2014,"Assume similar cost as for CH4 (g) submarine pipeline but with the same factor as between onland CH4 (g) pipeline and H2 (g) pipeline (2.86). This estimate is comparable to a 36in diameter pipeline calaculated based on d’Amore-Domenech et al (2021): 10.1016/j.apenergy.2021.116625 , supplementary material (=251 EUR/MW/km).",-
+H2 (g) submarine pipeline,investment,2002,545.325,EUR/MW/km,2014,"Implementation roadmap - Cross border projects and costs updates: https://ehb.eu/files/downloads/EHB-2023-20-Nov-FINAL-design.pdf Table 1","Same as for H2 (g) pipeline but with 7480 EUR/m"
 H2 (g) submarine pipeline,FOM,2015,3,%/year,2015,Assume same as for CH4 (g) submarine pipeline.,-
 H2 (g) submarine pipeline,lifetime,2015,30,years,2015,Assume same as for CH4 (g) submarine pipeline.,-
 H2 (g) submarine pipeline,electricity-input,2020,0.021,MW_e/1000km/MW_H2,2015,"Danish Energy Agency, Technology Data for Energy Transport (2021), Excel datasheet: H2 140.","Assumption for a 140 bar, 5-20 GW pipeline. Electric compression."
 H2 (g) submarine pipeline,electricity-input,2030,0.019,MW_e/1000km/MW_H2,2015,"Danish Energy Agency, Technology Data for Energy Transport (2021), Excel datasheet: H2 140.","Assumption for a 140 bar, 5-20 GW pipeline. Electric compression."
 H2 (g) submarine pipeline,electricity-input,2050,0.017,MW_e/1000km/MW_H2,2015,"Danish Energy Agency, Technology Data for Energy Transport (2021), Excel datasheet: H2 140.","Assumption for a 140 bar, 5-20 GW pipeline. Electric compression."
+H2 (g) submarine pipeline repurposed,investment,2002,191.48,EUR/MW/km,2014,"Implementation roadmap - Cross border projects and costs updates: https://ehb.eu/files/downloads/EHB-2023-20-Nov-FINAL-design.pdf Table 1","Same as for H2 (g) pipeline but with 1500 EUR/m"
+H2 (g) submarine pipeline repurposed,FOM,2015,3,%/year,2015,Assume same as for CH4 (g) submarine pipeline.,-
+H2 (g) submarine pipeline repurposed,lifetime,2015,30,years,2015,Assume same as for CH4 (g) submarine pipeline.,-
+H2 (g) submarine pipeline repurposed,electricity-input,2020,0.021,MW_e/1000km/MW_H2,2015,"Danish Energy Agency, Technology Data for Energy Transport (2021), Excel datasheet: H2 140.","Assumption for a 140 bar, 5-20 GW pipeline. Electric compression."
+H2 (g) submarine pipeline repurposed,electricity-input,2030,0.019,MW_e/1000km/MW_H2,2015,"Danish Energy Agency, Technology Data for Energy Transport (2021), Excel datasheet: H2 140.","Assumption for a 140 bar, 5-20 GW pipeline. Electric compression."
+H2 (g) submarine pipeline repurposed,electricity-input,2050,0.017,MW_e/1000km/MW_H2,2015,"Danish Energy Agency, Technology Data for Energy Transport (2021), Excel datasheet: H2 140.","Assumption for a 140 bar, 5-20 GW pipeline. Electric compression."
```

Comment:
In principle we could align these (and for other H2 pipeline technologies) also with the EHB:

434 MW_e / 16900 MW_H2 / 1000 km = 0.0257 MW_e/MW_H2 / 1000km

Very similar number. What do you thinkg

Reply:
Let's keep the way it is.